exporter/otelarrowexporter/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/otelar
 go 1.21
 
 require (
-	github.com/open-telemetry/otel-arrow v0.18.0
+	github.com/open-telemetry/otel-arrow v0.20.0
 	github.com/open-telemetry/otel-arrow/collector v0.18.0
```

Comment:
This seems to have missed an upgrade, this can be `github.com/open-telemetry/otel-arrow/collector v0.20.0` instead of `v0.18.0`